src/modules/auth/components/smart/Auth.tsx

```diff
@@ -1,136 +1,136 @@
-import React, { useState, useEffect } from 'react';
```

Comment:
you can rewrite export on the index file...

Reply:
I think this file is obselete now, just comment out to see if anything breaks